Evaluations_power_boxplots.ipynb

```diff
@@ -104,67 +129,5399 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-------------------- About to read transitions from server --------------------\n",
+      "Reading data for android phones\n",
+      "Loading transitions for phone ucb-sdb-android-1\n",
+      "About to retrieve messages using {'user': 'ucb-sdb-android-1', 'key_list': ['manual/evaluation_transition'], 'start_time': 1563606000, 'end_time': 1588204800}\n",
+      "response = <Response [200]>\n",
+      "Found 86 entries\n",
```

Comment:
How are we still getting `Response [200]` here? Shouldn't we be using the FileSpecDetails?